### DISCUSSION TITLE
blacksheep/sessions/

### DISCUSSION THREAD

**Discussion by giscus** [Neoteroi/BlackSheep-Docs#35] [Q&A]
# blacksheep/sessions/

The official documentation website of BlackSheep, a fast ASGI web framework for Python asyncio.

https://www.neoteroi.dev/blacksheep/sessions/

**Comment by hieulw** [comment]
so if I understand correctly, session here is stored in client instead of server ?

**Reply by RobertoPrevato** [reply]
Yes, it is correct.
